### DISCUSSION TITLE
using oncronjob after the new release announcement

### DISCUSSION THREAD

**Discussion by saicharan0124** [MetaMask/snaps#1206] [Q&A] [answered]
im gettting this error **"(0 , i.isCronjobSpecificationArray) is not a function"** what should i do to clear it ?

**Comment by ritave** [comment]
Hey @saicharan0124, I've created an issue in #1206.

Though more information about the error would be helpful. The stacktrace of the error, both from the console log and the background.html console log. Code to reproduce the issue. And anything more that we can use to help figure out what went wrong.

**Comment by FrederikBolding** [comment, marked as answer]
Hi @saicharan0124 !

Unfortunately a bug was shipped with the latest release of MetaMask Flask that breaks cronjobs entirely. We have already rectified the bug and it will be fixed as soon as we ship our next release.

Until then, if you are comfortable you can build MetaMask Flask locally by cloning the https://github.com/MetaMask/metamask-extension repository. You may also be able to use an older version of MetaMask Flask that does not have this bug.

**Reply by saicharan0124** [reply]
Thank u for the reply! Any idea when it would be fixed?

**Reply by FrederikBolding** [reply]
Hopefully we will be able to roll out a new release next week that fixes it 🤞
